documentation/2_Guiding_Principles.md

```diff
@@ -276,17 +276,15 @@ advantage for your own business and quite easy to do, since everything
 is well prepared. The [IDSA website provides](https://www.internationaldataspaces.org) all information. A hotline can help with
 questions (SupportOffice@internationaldatspaces.org).
 
-## Governance/legal framework
+## Data Space Governance Framework
 
-Relationship of data usage control and other types of control
-enforcement and legal agreements
+The Data Space Governance Framework is the defined set of policies, rules, and 
```

Comment:
policies are rules, so this is redundant. Is this rather policies, business process, and regulations... ?